diff --git a/config/BUILDCONFIG.gn b/config/BUILDCONFIG.gn
index bc4236a..9812564 100644
--- a/config/BUILDCONFIG.gn
+++ b/config/BUILDCONFIG.gn
@@ -262,7 +262,8 @@ _default_toolchain = ""
 
 if (target_os == "android") {
   # Targeting android on Mac is best-effort and not guaranteed to work.
-  assert(host_os == "linux", "Android builds are only supported on Linux.")
+  assert(host_os == "linux" || host_os == "mac",
+         "Android builds are only supported on Linux and (best-effort) Mac.")
   _default_toolchain = "//build/toolchain/android:android_clang_$target_cpu"
 } else if (target_os == "chromeos" || target_os == "linux") {
   # See comments in build/toolchain/cros/BUILD.gn about board compiles.
